### PR TITLE
add system paths to PKG_CONFIG_PATH

### PR DESCRIPTION
**Pull request by planthaber**
Ubuntu 16.04 seems not to inherit these

### CODE REVIEW

File: init.rb [hunk 1]
```diff
@@ -77,6 +77,8 @@
 
         Autobuild.env_add_path('PATH',File.join(release_install_dir,"bin"))
         Autobuild.env_add_path('CMAKE_PREFIX_PATH',release_install_dir)
+        Autobuild.env_add_path('PKG_CONFIG_PATH',File.join("/usr/share/","pkgconfig"))
+        Autobuild.env_add_path('PKG_CONFIG_PATH',File.join("/usr/lib/x86_64-linux-gnu/","pkgconfig"))
```
Comment (2maz) [inline init.rb]: better reuse the 'architecture' variable instead of hardcoding  x86_64-linux-gnu

**Comment by 2maz** [comment]
To what does autoproj set  the PKG_CONFIG_PATH?

**Comment by planthaber** [comment]
It does not include these folders, resultin in e.g. "eigen3 cannot be found"

**Comment by planthaber** [comment]
now i've successfully build my buildconf on Ubuntu 16.04

**Comment by 2maz** [comment]
could you squash your commits

**Comment by planthaber** [comment]
squashed

**Comment by 2maz** [comment]
Thx

**Pull request merged**